packages/ses/src/ses.d.ts

```diff
@@ -0,0 +1,38 @@
+type safetyPreference = 'safe' | 'unsafe';
+
+type LockdownOpts = {
+  fringeSet?: {
+    add?: Function,
+    has?: Function,
+  },
+  dateTaming?: safetyPreference,
+  errorTaming?: safetyPreference,
+  mathTaming?: safetyPreference,
+  regExpTaming?: safetyPreference,
+
+  [extraOption:string]?: any,
+};
+
+declare var lockdown = (LockdownOpts) => void;
+
+declare var harden = (any) => any;
+
+declare class Compartment {
+  constructor(endowments: Object);
```

Comment:
```ts
constructor(endowments: Object, modules: ModuleMap, CompartmentConstructorOptions)
```

Some TS from here might be useful: https://github.com/tc39/proposal-compartments#sketch